src/features/email/component/email-view/email-view-response-more.tsx

```diff
@@ -0,0 +1,79 @@
+import { ChevronDown, Forward, Reply, ReplyAll, Trash2 } from 'lucide-react';
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuTrigger,
+} from 'components/ui/dropdown-menu';
+import { TEmail } from '../../types/email';
+import { Button } from 'components/ui/button';
+import AvatarFrame from 'components/ui/avatar-frame';
+
+interface EmailViewResponseTypeProps {
+  selectedEmail?: TEmail;
+  isReply: boolean;
+  setIsReply: (isReply: boolean) => void;
+}
+
+const EmailViewResponseMore = ({
+  isReply,
+  setIsReply,
+  selectedEmail,
+}: EmailViewResponseTypeProps) => {
+  const onReplyClick = () => {
+    setIsReply(!isReply);
+  };
+
+  return (
+    <>
+      <div className="border-b border-low-emphasis py-1">
+        <div className="flex gap-2 items-center ">
+          <div className="flex  gap-2 text-medium-emphasis ">
+            <Reply className="h-5 w-5 " />
+
+            <DropdownMenu>
+              <DropdownMenuTrigger asChild>
+                <ChevronDown className="h-5 w-5 cursor-pointer" />
+              </DropdownMenuTrigger>
+              <DropdownMenuContent className="w-52">
```

Comment:
Use min-w-52 instead of w-52, so that if you need to add content longer than w-52, it remains flexible. Restricting the width with w-52 would prevent that, but min-w-52 allows the element to expand as needed.